Changes.md

```diff
@@ -4,6 +4,10 @@ This file is a manually maintained list of changes for each release. Feel free
 to add your changes here when sending pull requests. Also send corrections if
 you spot any mistakes.
 
+## Unreleased
```

Comment:
I appreciate taking the time to do this particular change. If you want to, then can we match all previous commits? https://github.com/felixge/node-require-all/commit/a4efa8b4eaace1bf1ad4ce102a1a705925ac1363 for instance.

Reply:
done